### ISSUE TITLE
Remove OIDC return_uri hack

### ISSUE THREAD

**Issue by mkosek** [freeipa/freeipa-wiki#4]
As FreeIPA wiki is running behind proxy, it's internal URI is different from the exposed one. I had to hack OIDC plugin to return a configured return URI, otherwise Ipsilon would reject the OIDC authentication: cc326520da51bba95026c1fb1051a695d959142a.

This hack should be fixed, otherwise the OIDC plugin needs to be hacked again after every update.

**Comment by simo5** [comment]
Can you upstrem a configuration setting to the plugin, so that everyone benefits from it and we do not have to hack it every time ? Running behind reverse proxies is pretty common these days.

**Comment by mkosek** [comment]
Related change that works with vendor being built dynamically: 1479e11179f2f0d84423f09f85a32f2cfb1448f2

**Comment by mkosek** [comment]
@simo5, yes, that's my plan actually. This is why I created this ticket, to remind me/us to remove this hack!
